communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,74 @@
+
+See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
+
+## Acknowledgements and celebrations!
+
+Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.
+
+Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and The Turing Way at the [RSEconUK 2019](https://rse.ac.uk/conf2019/). She shared her training materials [online on Zenodo](https://zenodo.org/record/3404774#.XZN7h5MzbpB), under the DOI: 10.5281/zenodo.3404774.
+
+Rachael, Community Manager of Software Sustainability Institute, attended the Open Science Fair 2019 in Porto, Portugal, last month, where she presented a poster and a [demo session](https://www.opensciencefair.eu/demos-2019/the-turing-way-a-handbook-for-reproducible-data-science) on The Turing Way project. See her [tweet-thread](https://twitter.com/rachaelevelyn/status/1173542269355532289) from the conference.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Hi <a href="https://twitter.com/hashtag/OSFair2019?src=hash&amp;ref_src=twsrc%5Etfw">#OSFair2019</a>! 👋 I’ve got a poster presenting the <a href="https://twitter.com/hashtag/TuringWay?src=hash&amp;ref_src=twsrc%5Etfw">#TuringWay</a>: an open &amp; collaborative project to develop a handbook for reproducible data science. Feel free to come chat to me about this project (or anything <a href="https://twitter.com/hashtag/OpenScience?src=hash&amp;ref_src=twsrc%5Etfw">#OpenScience</a> related!) at poster stand #80. <a href="https://t.co/tMAXVn2mKq">pic.twitter.com/tMAXVn2mKq</a></p>&mdash; Dr. Rachael Ainsworth (@rachaelevelyn) <a href="https://twitter.com/rachaelevelyn/status/1173625329686110208?ref_src=twsrc%5Etfw">September 16, 2019</a></blockquote> 
+
+## Connect with us!
+
+- [About the project](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
+- [The Turing Way Book](https://the-turing-way.netlify.com)
+- [GitHub](https://github.com/alan-turing-institute/the-turing-way)
+- [Gitter](https://gitter.im/alan-turing-institute/the-turing-way)
+- [YouTube Videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)
+- Twitter Hashtag [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)
+
+You are welcome to contribute contents for the next newsletter by by
+emailing the project coordinator [Kirstie Whitaker](mailto:kwhitaker@turing.ac.uk).
```

Comment:
Ah - this is not a great mechanism - I'm terrible with email 😬 

Can they email you @malvikasharan? Or can we make an issue?

Fine to leave it as me if that's the best option....

Reply:
Yes, of course :) They can email me in my turing email id, msharan@turing.ac.uk

Reply:
Thank you so much 🙏

Reply:
Updated the newsletter with your comments. Do you have any comment on the format?